Admin/Infrastructure/Proxmox/2-ChangingHostNames.md

```diff
@@ -1,8 +1,8 @@
 # Changing Host Names
 
-Author(s):
-    Chisom (Ask me anything, don't bother Luke),
-    Luke
+*Author(s):*

```

Comment:
Similar Suggestion to the one in the `3-Removing Nodes` document.

```suggestion
**Author(s)**: 
* Chisom
* Luke
---
```
* This way "Authors: is bolded 
* The Author's name is indented with a bullet point (Could omit the bullet point)
* The `---` inserts a line